app/models/pawn.rb

```diff
@@ -1,5 +1,75 @@
 class Pawn < Piece
 
+  def valid_move?(new_x, new_y)
+    update_moved_status!
+    position_exist?(new_x, new_y) && passes_pawn_rules?(new_x, new_y)
+  end
+
+  def passes_pawn_rules?(x, y)
+    return true if two_squares_forward?(x, y)
+    return true if one_square_forward?(x, y)
+    return true if diagonal_move?(x, y)
+
+    return false
+  end
+
```

Comment:
lets add a 'private' keyword here to indicate that the following methods are private